HYUNSANG_OH/app.js

```diff
@@ -4,9 +4,14 @@ const cors = require("cors");
 const morgan = require("morgan");
 const dotenv = require("dotenv");
 const { DataSource } = require("typeorm");
-
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
 dotenv.config();
 
+//암호화 관련 코드
+const secretKey = process.env.secretKey;
```

Comment:
```suggestion
const secretKey = process.env.SECRET_KEY;
```
기존에 환경 변수 네이밍 컨벤션과 맞춰서 다시 작성해 주세요~